www/migrate.spt

```diff
@@ -360,7 +360,7 @@ title = _("Migrating from Gratipay")
                     "{0} is already connected to a different Liberapay account.",
                     existing_account.email
                 ) }}</div>
-                <p>{{ _("If this address belongs to you please log in before continuing:") }}</p>
+                <p>{{ _("If this address belongs, to you please log in before continuing:") }}</p>
```

Comment:
This comma isn't in the correct position.

Reply:
Fixed. My `sed` command placed the comma well in the `.po` files, so there is need to execute that command again.